cyclus_input_gen/templates.py

```diff
@@ -13,7 +13,7 @@ class template_collections:
       <Reactor>
         <fuel_inrecipes>  <val>natl_u_recipe</val>       </fuel_inrecipes>
         <fuel_outrecipes> <val>used_candu_recipe</val>  </fuel_outrecipes>
-        <fuel_incommods>  <val>natl_u</val>                   </fuel_incommods>
+        <fuel_incommods>  <val>candu_fuel</val>                   </fuel_incommods>
```

Comment:
interesting change... given there are no candus in this simulation, what is the purpose of this change at this moment?

Reply:
This should have been used, I forgot to remove the comment on the PHWR reactor generation. There are supposed to be 3 PHWRs in Romania. I will fix that.